### PR TITLE
ns querying -> galois; more specific class names

### PR DESCRIPTION
**Pull request by l-hoang**
Change querying namespace to galois for consistency. also renamed some
of the classes in the querying namespace to be more descriptive of what
they are so that they do not conflict in the future.

### CODE REVIEW

File: libquery/include/galois/graphs/AttributedGraph.h [hunk 1]
```diff
@@ -178,8 +178,8 @@ class AttributedGraph {
    * TODO break this big function into modular pieces (e.g., query graph
    * building vs querying proper)
    */
-  size_t matchQuery(std::vector<querying::CompilerNode>& queryNodes,
-                    std::vector<querying::CompilerEdge>& queryEdges,
+  size_t matchQuery(std::vector<galois::CypherCompilerNode>& queryNodes,
+                    std::vector<galois::CypherCompilerEdge>& queryEdges,
```
Comment (roshandathathri) [inline libquery/include/galois/graphs/AttributedGraph.h:182]: QueryNode and QueryEdge would be simpler and better names in my opinion.

File: libquery/include/querying/CypherCompiler.h [hunk 2]
```diff
@@ -56,7 +56,7 @@ struct ReturnResult {
   bool is_count;
 
   //! Default constructor initializes everything to null
-  ReturnResult()
+  CypherReturnResult()
```
Comment (roshandathathri) [inline libquery/include/querying/CypherCompiler.h:59]: QueryResult would be simpler and better in my opinion

File: libquery/include/galois/graphs/AttributedGraph.h (hunk 1 above)
Comment (l-hoang) [reply libquery/include/galois/graphs/AttributedGraph.h:182]: IIRC, those names conflict with the names of the data structures being used for the QueryGraph structure. CompilerQueryNode/CompilerQueryEdge work? (the only place this struct is used is in the compiler, so I want to keep the Compiler tagz0.

Comment (roshandathathri) [reply libquery/include/galois/graphs/AttributedGraph.h:182]: Sounds good.

File: libquery/include/querying/CypherCompiler.h (hunk 2 above)
Comment (l-hoang) [reply libquery/include/querying/CypherCompiler.h]: Decided to go with CompilerQueryResult for consistency with the node/edge. Wordy, but people won't mistakenly use it elsewhere outside the compiler.

**Pull request merged**